svm-builds/build.rs

```diff
@@ -1,3 +1,5 @@
+#![allow(dead_code)]
```

Comment:
why is this needed?

Reply:
The `generate` function is unused when the `_offline` feature is enabled, which generates a lot of warnings